packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -98,9 +98,17 @@ export class ColorPicker
    * When `true`, an empty color (`null`) will be allowed as a `value`.
    *
    * When `false`, a color value is enforced, and clearing the input or blurring will restore the last valid `value`.
+   *
+   * @deprecated Use `clearable` instead
    */
   @Prop({ reflect: true }) allowEmpty = false;
 
+  @Watch("allowEmpty")
+  @Watch("clearable")
+  handleAllowNullValues(): void {
```

Comment:
I'd like to avoid using "null values" and instead use a name for the watcher that matches the watched properties. Some suggestions:

* `handleAllowClearingPropChange`
* `handleAllowEmptyOrClearableChange`